app/assets/stylesheets/sephora_style_guide/variables/_card.scss

```
@@ -14,7 +14,7 @@ $card-disabled-color:        $grey-mid;
 
 $card-img-overlay-padding:  16.25px !default;
 
-$card-deck-margin:          (map-get($grid-gutter-widths, sm) / 2) !default;
+$card-deck-margin:          calc(map-get($grid-gutter-widths, sm) / 2) !default;
```

question: this one won't be error meh?
I thought should be:
```suggestion
$card-deck-margin:          calc(#{map-get($grid-gutter-widths, sm) / 2)} !default;
```